database/schema.sql

```diff
@@ -5,3 +5,41 @@ set client_min_messages to warning;
 drop schema "public" cascade;
 
 create schema "public";
+
+CREATE TABLE "games" (
+	"gameId" serial NOT NULL,
+	"createdAt" timestamp with time zone NOT NULL default now(),
+	CONSTRAINT "games_pk" PRIMARY KEY ("gameId")
+) WITH (
+  OIDS=FALSE
+);
+
+
+
+CREATE TABLE "moves" (
+	"gameId" integer NOT NULL,
+	"start" integer NOT NULL,
+	"end" integer NOT NULL,
+	"moveId" serial NOT NULL,
+	CONSTRAINT "moves_pk" PRIMARY KEY ("moveId")
+) WITH (
+  OIDS=FALSE
+);
+
+
+
+CREATE TABLE "postedGames" (
+	"postId" serial NOT NULL,
+	"playerName" TEXT NOT NULL,
+	"message" TEXT NOT NULL,
+	"playerSide" TEXT NOT NULL,
+	"createdAt" timestamp with time zone NOT NULL default now(),
+	CONSTRAINT "postedGames_pk" PRIMARY KEY ("postId")
```

Comment:
What is the difference between a `"postedGame"` and a `"game"`? Can we combine these?